spec/swagger_helper.rb

```diff
@@ -204,6 +204,16 @@
                          crossposted_at published_at last_comment_at published_timestamp user
                          reading_time_minutes]
           },
+          FollowedTag: {
+            description: "Representation of a followed tag",
+            type: "object",
+            properties: {
+              id: { description: "Tag id", type: :integer, format: :int64 },
+              name: { type: :string },
+              points: { type: :number, format: :float }
```

Comment:
(suggestion, non-blocking): I think it could be useful to add a description to points as its not obvious what this parameter is.